docker/docker-compose.yml

```diff
@@ -51,6 +51,8 @@ services:
 
     env_file:
       - ./locale_env.sh
-    command: bash -c "pip3 install setuptools --upgrade && python3 setup.py clean && python3 setup.py install --react-interface=/dist && /api/wait-for-postgres.sh pg gunicorn --paster /api/docker/docker.ini"
```

Comment:
Why "pip3 install setuptools" command was deleted?

Reply:
That's actually like it is in the current heavy_refactor master.
And yeah, why is pull against corpus_markup if we are going to merge it into master anyway?

Reply:
> Why "pip3 install setuptools" command was deleted?

That's because this command is been also specified in Dockerfile